app/scripts/modules/tencentcloud/src/pipeline/stages/shrinkCluster/shrinkClusterStage.ts

```diff
@@ -0,0 +1,11 @@
+import { validate } from './shrinkClusterValidators';
+import { Registry, IStageTypeConfig } from '@spinnaker/core';
+import { ShrinkClusterConfig } from './ShrinkClusterConfig';
+
+Registry.pipeline.registerStage({
+  provides: 'shrinkCluster',
+  key: 'shrinkCluster',
+  cloudProvider: 'tencentcloud',
+  component: ShrinkClusterConfig,
+  validateFn: validate,
+} as IStageTypeConfig);
```

Comment:
```suggestion
});
```
I think we just missed one of these `as IStageTypeConfig` to remove.